Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -361,6 +361,14 @@ final class ConfigGenerator: ConfigGenerating {
             }
         }
 
+        if !target.odrInitialInstallTags.isEmpty {
+            settings["ON_DEMAND_RESOURCES_INITIAL_INSTALL_TAGS"] = .string(target.odrInitialInstallTags.joined(separator: " "))
```

Comment:
TODO: We should sort Initial Install Tags

Reply:
Also, tags names can contains spaces and other special characters.
Based on fact that Initial Install tags and Prefetch Order Tags are presented in `pbxproj` as single strings of tags separated by space character, we have to decode spaces and probably some other symbols. Just like how Xcode does it.